### PR TITLE
Test Network Data

### PR DESCRIPTION
**Pull request by prudmiry**
Test Network Data

Test Network Statistics
Test LRO
Test GRO
Test Big Ping
Test Network Interface

Signed-off-by: prudhvi <mprudhvi@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by prudmiry** [comment]
Please look into job log file
[job.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/471372/job.txt)

File: io/net/net_data.py.data/net_data.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Interfaces:
+    Interface1:
+        peerip: "10.10.10.11"
+        iface: "enP2p1s0f4"
+
```
Comment (PraveenPenguin) [inline io/net/net_data.py.data/net_data.yaml]: please remove this white space , please rebase your commit. than force update

**Comment by prudmiry** [comment]
Modified and pushed again @PraveenPenguin

**Comment by PraveenPenguin** [comment]
@prudmiry thanks , looks good to me

**Pull request merged**